### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by doudou**
This is the initial code, not tested against any device.

### CODE REVIEW

File: src/RTU.cpp [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    int duration_us = ceil(1000000.0 / static_cast<double>(bitrate) *
+                           SERIAL_BITS_PER_CHAR * 3.5);
+    return base::Time::fromMicroseconds(max(duration_us, 1750));
+}
+
+uint8_t* RTU::formatFrame(uint8_t* buffer, int address, int functionID,
+                     std::vector<uint8_t> const& payload) {
+
+    return RTU::formatFrame(buffer, address, functionID,
+                            &(*payload.begin()), &(*payload.end()));
```
Comment (g-arjones) [inline src/RTU.cpp]: Dereferencing the end iterator is undefined behavior

File: src/RTU.hpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+#include <modbus/Frame.hpp>
+#include <array>
+
+namespace modbus {
+    /**
+     * Implementation of the Modbus RTU (binary) protocol
+     */
+    namespace RTU {
+        /** Modbus broadcast address */
+        static const int BROADCAST = 0;
```
Comment (g-arjones) [inline src/RTU.hpp:15]: enable c++11

**Review by g-arjones — APPROVED** [review]

File: src/RTU.cpp (hunk 1 above)
Comment (doudou) [reply src/RTU.cpp]: :+1:

File: src/RTU.hpp (hunk 2 above)
Comment (doudou) [reply src/RTU.hpp:15]: Done

**Review by g-arjones — APPROVED** [review]

**Pull request merged**